frontend/src/lib/components/CommandBar/constants.ts

```diff
@@ -40,7 +41,7 @@ export const tabToName: Record<Tab, string | null> = {
     [Tab.Action]: 'Actions',
     [Tab.Cohort]: 'Cohorts',
     [Tab.Dashboard]: 'Dashboards',
-    [Tab.EventDefinition]: 'Event definitions',
```

Comment:
We're calling the event definitions "events" elsewhere, thus changing it here as well.